### DISCUSSION TITLE
[Bug report] GET 502 error when trying to load more changes in the files changed tab of a PR

### DISCUSSION THREAD

**Discussion by ManlyMarco** [community/community#6938] [Pull Requests]
Only first 3 files are loaded. When trying to view more of the files changed, I get a GET 502 error in console and it never finishes. This happens only when I'm logged in, when logged out changes load without issue. Because of this I'm unable to add suggested code changes to batch (and I really need to batch them because of the large amount).

Screenshot of it happening: ![image](https://user-images.githubusercontent.com/39247311/139326924-3d8ed093-ef78-4887-8205-7c7594d13241.png)

There are many conversations started in this PR and this seems to be the issue. From our experience having many conversations that reference code lines makes the website really struggle, but this is the first time it straight up fails.

Tested in latest versions of Opera and Edge, there's no difference between the two.

PR in question https://github.com/IllusionMods/KKS-Translation/pull/113/files

**Comment by fabien-somnier** [comment]
I'm experiencing the same issue today (on Chrome)

**Reply by cosminbosutar** [reply]
I have the same problem in Edge

**Reply by udipl** [reply]
I have the same problem, and my PR has many (~200) changed files and many commits, but almost no conversations.

**Reply by wbrowar** [reply]
Having the same issue and it doesn't seem to be browser specific. I had this occur in Safari and Edge.

**Reply by udipl** [reply]
I would be very surprised if a 502 error were browser-specific.

**Reply by cosminbosutar** [reply]
@udipl @wbrowar I only had this error for 1 day and ] the next day everything worked fine.

I haven't done anything to solve the error....I just waited it out.

Maybe they are deploying something?

Hope this helps

**Reply by wbrowar** [reply]
@cosminbosutar Thanks for sharing that. I just checked and it seems to be working again, so maybe you're right that it's some sort of deployment hiccup or something. 🤷🏻‍♂️

**Reply by udipl** [reply]
I just checked again and my issue was also resolved. I'm glad it was addressed.

**Reply by benvogler** [reply]
I still have this issue frequently on large PRs
